src/loaders/gltf.rs

```diff
@@ -0,0 +1,25 @@
+use std::rc::Rc;
+
+use crate::geometry::Mesh;
+use crate::material::Material;
+
+pub fn load_file(filepath: &str) -> Vec<Mesh> {
+    let (document, buffers, _) = gltf::import(filepath).expect("Could not open Gltf file");
```

Comment:
As we refactor this code to be more robust, we should replace a lot of these calls to `expect` and a lot of our panics (e.g. in `assert_eq`) with proper error handling using `Result`. **No need to do this now.** We'll go one step at a time. :)

Fun fact: Most Rust developers I know actually develop exactly like this. They litter their code with unwrap/panic until they get it work. Then they go back and add proper error handling so they can produce nice error messages for their users.